cumulus/polkadot-parachain/polkadot-parachain-lib/src/cli.rs

```diff
@@ -119,6 +119,14 @@ pub struct Cli<Config: CliConfig> {
 	#[command(flatten)]
 	pub run: cumulus_client_cli::RunCmd,
 
+	/// Start a dev node that produces a block each `dev_block_time` ms.
+	///
+	/// This is a dev option, and it won't result in starting or connecting to a parachain network.
+	/// The resulting node will work on its own, running the wasm blob and artificially producing
+	/// a block each `dev_block_time` ms, as if it was part of a parachain.
+	#[arg(long)]
+	pub dev_block_time: Option<u64>,
```

Comment:
I now wonder if the difference between `--dev-block-time` and `--dev` is clear? 

Perhaps nitpicking, and ofc this can be fixed with great documentation.

cc @skunert

Reply:
First glance, what comes to mind is a subcommand here that doesn't inherit all the other `SharedParams` and such, but I think that has already been explored?

Reply:
Yes, a separate subcommand was explored before, but I admit I didn't think about this issue back then. But even if we add a separate subcommand, from what I understand it still needs to contain the `SharedParams` in order to be able to create the runner: 

```
fn create_runner<T: CliConfiguration<DVC>, DVC: DefaultConfigurationValues>(
		&self,
		command: &T,
	) -> Result<Runner<Self>>

...

pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
	/// Get the SharedParams for this object
	fn shared_params(&self) -> &SharedParams;
       
       ...
}
```

Anyway, it is a good point. But could we explore it in a separate issue ? It would be good if we could merge this one before doing the renamings in https://github.com/paritytech/polkadot-sdk/issues/5566 and in general before doing other changes to `polkadot-parachain-lib/bin` in order to avoid merging conflicts.

Reply:
Opened separate issue for this as discussed offline: https://github.com/paritytech/polkadot-sdk/issues/5903